src/libaktualizr/package_manager/androidmanager.h

```diff
@@ -0,0 +1,43 @@
+#ifndef ANDROIDMANAGER_H
+#define ANDROIDMANAGER_H
+
+#include "package_manager/packagemanagerinterface.h"
+#include "storage/invstorage.h"
+
+class AndroidInstallationDispatcher;
+
+class AndroidManager : public PackageManagerInterface {
+ public:
+  explicit AndroidManager(std::shared_ptr<INvStorage> storage) : storage_(std::move(storage)) {}
+  ~AndroidManager() override = default;
+  std::string name() const override { return "android"; }
+  Json::Value getInstalledPackages() const override;
+
+  Uptane::Target getCurrent() const override;
+  bool imageUpdated() override { return true; };
+
+  data::InstallOutcome install(const Uptane::Target &target) const override;
+  data::InstallOutcome finalizeInstall(const Uptane::Target &target) const override {
+    (void)target;
+    throw std::runtime_error("Unimplemented");
+  }
```

Comment:
We might want to just return a positive result instead of throwing if there really is nothing to do here. This function exists for the case when installation requires reboot or some external action before we can really call it complete. Does that apply to the android case?

Reply:
I did android package manager when finalizeInstall call was not yet introduced. It may leads some changes in the current implementation, I need to check.